doc/tutorials/active_dendrite/nestml_active_dendrite_tutorial.ipynb

```diff
@@ -0,0 +1,595 @@
+      "text/plain": [
+       "<Figure size 600x400 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "evaluate_neuron(\"iaf_psc_exp_active_dendrite_nestml\", neuron_parms={\"I_th\": 100., \"I_dAP_peak\": 400.})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the top panel, we can see the synaptic and dendritic action potential currents separately. Incoming action potentials from the presynaptic partner, triggering postsynaptic currents, are indicated by orange triangles. The middle panel shows the total synaptic current, which is equal to the sum of synaptic and dendritic action potential current. The bottom panel shows the resulting postsynaptic membrane potential, and postsynaptic (somatic) action potentials using orange triangles.\n",
```

Comment:
what about using the dAP abbreviation here, so instead of "dendritic action potential currents" you'd write "dAP currents", which makes it a bit easier to read. 
As I told you last time, for me dAP current is also part of the synaptic current `I_syn`. But, this is a matter of definition and it's up to you to choose between the two terminologies.